### DISCUSSION TITLE
opgg大乱斗的符文怎么只有2个了

### DISCUSSION THREAD

**Discussion by steyyynoob** [cangzhang/ChampR#211] [General]
前几天还是很多个，这几天只有两个了，不知道咋回事

**Comment by cangzhang** [comment]
官网上每个英雄推荐符文只有 **2 个**. 
其余的符文用的相对较少, 没有参考意义

<img width="1143" alt="image" src="https://user-images.githubusercontent.com/1357073/188553642-8b366348-a795-4460-8441-499d4cf07228.png">

**Reply by steyyynoob** [reply]
runes那一页有4个，不方便的话就两个也够用了
